docs/src/main/paradox/docs/delta/api/current/views/aggregated-sparql-view-api.md

```diff
@@ -72,8 +226,19 @@ The `Content-Type` HTTP header for POST request is `application/sparql-query`.
 **Example**
 
 Request
-:   @@snip [sparql-view-search.sh](../assets/views/sparql-view-search.sh)
+:   @@snip [search.sh](../assets/views/blazegraph/search.sh)
 
 Response
-:   @@snip [sparql-view-search.json](../assets/views/sparql-view-search.json)
+:   @@snip [search.json](../assets/views/blazegraph/search.json)
+
+@@@ note { .tip title="Response formats" }
+
+From Delta 1.5, we have added support for multiple Content Negotiation types when querying the SPARQL view, allowing clients to request different response formats. The Content Negotiation is controlled by the HTTP `Accept` header. The following values are supported:
```

Comment:
I don't think we need this in a `note` block but just in the previous paragraph where says `The Content-Type HTTP Header`.

Just adding block notes on every release is not going to make the documentation more understandable.